src/components/FbxDropdown/FbxDropdown.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="drop-down">
+    <button class="drop-down-button" @click="dropDownButtonClicked" @focusout="clickedOutside">
+      {{ menuText }}
+      <span v-if="menuText" class="nbsp"></span>
+      <span class="menu-icon fbx-icon-arrow-down"></span>
+    </button>
+    <transition name="slide-down">
+      <div class="drop-down-content" v-if="isMenuOpen">
+        <slot></slot>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+const delayForClose = 200
```

Comment:
Should we use caps + snake case for constants? Down below, in the `setTimeout`, I thought `delayForClose` was a callback or something 😄
```suggestion
const CLOSE_DELAY = 200
```